### DISCUSSION TITLE
Chromiecraft

### DISCUSSION THREAD

**Discussion by Winterscar77** [chromiecraft/chromiecraft#7446] [General]
Hello,for some time i am playing on your server. Until yesterday everything was fine, from yesterday i have freezes and i need to reset my pc. I even installed the 4 giga patch memory and  still i have the problem. I cannot play it anymore. Can you help me? It is from a recent patch? The players from official server have the same problem when a major patch is installed.

**Comment by Nyeriah** [comment]
Where in the world is your character when you experience those freezes?
